spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -39,6 +39,9 @@
   end
 
   describe "analyze method" do
+    require "reek"
+    require "reek/cli/application"
```

Comment:
might be better to refactor the [`run!` method in the generator](https://github.com/metricfu/metric_fu/blob/115c5f1a0d906af49529c0b67846e71127d9f808/lib/metric_fu/metrics/reek/generator.rb#L18-L24) so we can use this logic just the once to get the examiner and small classes...